operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -69,7 +69,7 @@ type Egress struct {
 }
 
 // ConnectivityPolicy is a type for values of spec.egress.connectivityPolicy.
-//+kubebuilder:validation:Enum=Online;Offline
+// +kubebuilder:validation:Enum=Online;Offline
```

Comment:
more of a super nit: Sometimes its `// +kubebuilder...` or `//+kubebuilder`.  Would be great to be consistent now that we change it here / or just _not_ change this.

Reply:
I didn't change these by hand. I don't why know, but the formatter is behaving very strangely...

Reply:
So I finally figured it out. I get a different formatting result depending on whether I use go1.18 (`//+kubebuilder`) vs go1.19 (`// +kubebuilder`).